authz/sdk_server_interceptors.go

```diff
@@ -62,6 +62,7 @@ func (i *StaticInterceptor) UnaryInterceptor(ctx context.Context, req interface{
 	err := i.engines.IsAuthorized(ctx)
 	if err != nil {
 		if status.Code(err) == codes.PermissionDenied {
+			logger.Infof("%v", err)
```

Comment:
Can you add more info to the log entry?
And maybe `v(2)`?

Reply:
Sure. PTAL